src/engine/strat_engine/liminal/identify.rs

```diff
@@ -171,30 +176,81 @@ impl DeviceInfo {
     }
 }
 
-impl From<StratBlockDev> for DeviceInfo {
+impl From<StratBlockDev> for Vec<DeviceInfo> {
     fn from(bd: StratBlockDev) -> Self {
+        fn is_valid(blockdev: &StratBlockDev) -> StratisResult<(bool, bool)> {
```

Comment:
I think for this is_valid() method it would be helpful to put in a header comment that documents what it is guarding against by using the blkid probes.